content/en/developers/guide/unified-tagging-advanced-usage.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Unified Tagging Advanced Usage Guide
+kind: guide
+aliases:
+  - /developers/guide/unified-tagging-advanced-usage
+---
+
+## Overview
+
+This guide suggests ways to configure and migrate to [unified service tagging][1] based on specific use cases.
+
+## Using Custom Tags
+
+You can continue to use the `env` `service` and `version` tags as they are configured for unified service tagging. However, if you'd like to achieve a unified tagging experience with your own custom tags, there are some options available to you:
+
+**Note**: While some products support arbitrary tags, others have more specific expectations. As such, navigation between products might be difficult if one data source has a tag that another does not have or does not support.
```

Comment:
It's a little confusing that there is a trailing `:` at the end of this sentence. Maybe rephrase as `there are several options available which are listed below.` or similar. @sarina-dd @jdgumz thoughts?